test/index.test.js

```diff
@@ -1206,6 +1206,60 @@ Previous tag: ''
           expect.assertions(1)
         })
       })
+
+      describe('with forked pull request', () => {
+        it('exclude forked pull requests', async () => {
+          getConfigMock()
+
+          nock('https://api.github.com')
+            .get('/repos/toolmantim/release-drafter-test-project/releases')
+            .query(true)
+            .reply(200, [require('./fixtures/release')])
+
+          nock('https://api.github.com')
+            .post('/graphql', (body) =>
+              body.query.includes('query findCommitsWithAssociatedPullRequests')
+            )
+            .reply(
+              200,
+              require('./fixtures/__generated__/graphql-commits-forking.json')
+            )
+
+          nock('https://api.github.com')
+            .post(
+              '/repos/toolmantim/release-drafter-test-project/releases',
+              (body) => {
+                expect(body).toMatchInlineSnapshot(`
+                  Object {
+                    "body": "# What's Changed
+
+                  * Add documentation (#28) @jetersen
+                  * Update dependencies (#27) @jetersen
+                  * Bug fixes (#25) @jetersen
+                  * Add big feature (#24) @jetersen
+                  * Add alien technology (#23) @jetersen
+                  * Add documentation (#5) @TimonVS
+                  * Update dependencies (#4) @TimonVS
```

Comment:
something went wrong when I created the branch
However the PR with the title `🙅‍♂️ Forking PR 🙅‍♂️` is filtered out